wolfSSL/src/chapter02.md

```diff
@@ -576,6 +914,42 @@ Turns on the use of ARMv8 hardware acceleration.
 
 Turns on fast math RSA non-blocking support for splitting RSA operations into smaller chunks of work. Feature is enabled by calling [`wc_RsaSetNonBlock()`](group__RSA.md#function-wc_rsasetnonblock) and checking for `FP_WOULDBLOCK` return code.
 
+#### WC_RNG_TYPE_DEFINED
+
+Used as a guard on redeclaration.
+
+#### WOLFSSL_DH_TYPE_DEFINED
+
+Used as redeclaration guard for DH.
+
+#### WOLFSSL_DSA_TYPE_DEFINED
+
+Used as redeclaration guard for DSA.
+
+#### WOLFSSL_EC_TYPE_DEFINED
+
+Used as redeclaration guard for EC.
+
+#### WOLFSSL_RSA_TYPE_DEFINED
+
+Used as redeclaration guard for RSA.
+
+#### WOLFSSL_ECDSA_TYPE_DEFINED
+
+Used as redeclaration guard for ECDSA.
+
+#### WOLFSSL_DTLS_EXPORT_TYPES
+
+Used as a macro guard for redefinition warning.
+
+#### WC_RSA_BLINDING
+
+Used to enable timing resistance.
+
+#### WC_RSA_PSS
+
+TLS 1.3, in some cases, requires RSA PSS padding support.
```

Comment:
Enables RSA PSS padding. The only TLS v1.3 RSA padding scheme supported is PSS (per specification). PSS padding uses random padding.